picture-stealer/src/main/java/com/bobocode/picturestealer/entity/PhotoEntity.java

```diff
@@ -0,0 +1,65 @@
+package com.bobocode.picturestealer.entity;
+
+import jakarta.persistence.Cacheable;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.persistence.Version;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+/**
+ * @author "Maksym Oliinyk"
+ */
+@Getter
+@NoArgsConstructor
+@EqualsAndHashCode(exclude = "camera")
+@ToString(onlyExplicitlyIncluded = true)
+@Cacheable
```

Comment:
Caching looks like an overhead for this task too.

Reply:
This is used to cache the state of an entity. Query Cache doesn't cache the state of the entities or the actual data. Instead, it caches the identifiers (primary keys) of the entities returned by a query. When a cached query is executed again, Hibernate uses these identifiers to retrieve the entities. If these entities are in the Second-Level Cache, they are fetched from there; otherwise, a database query is made.So it should always be used in conjunction with the second-level cache.